Doc 4856 fgac (#16691)
cockroachdb/docs: cockroachcloud/connect-to-a-serverless-cluster.md

## Before you start

- [Create a {{ site.data.products.serverless }} cluster](create-a-serverless-cluster.html).

> - _(Optional)_ [Create a new SQL user](managing-access.html#create-a-sql-user).

Reviewer: ```suggestion
- (Optional) [Create a new SQL user](managing-access.html#create-a-sql-user).
```

{% include cockroachcloud/postgresql-special-characters.md %}

{{site.data.alerts.callout_info}}

> If you forget your SQL user's password, an [org admin](authorization.html#org-administrator-legacy) can change the password on the **SQL Users** page.

Reviewer: ```suggestion
An [org admin](authorization.html#org-administrator-legacy) can reset a SQL user's password on the [**SQL Users** page](../cockroachcloud/user-authorization#change-a-users-password).
```
